features/admin.rules.v1/components/rule-conditions.tsx

```diff
@@ -383,26 +389,36 @@ const RuleConditions: FunctionComponent<RulesComponentPropsInterface> = ({
     }: ResourceListSelectProps) => {
 
         const [ resourceDetails, setResourceDetails ] = useState<ResourceInterface>(null);
-        const valueReferenceAttribute: string = findMetaValuesAgainst?.value?.valueReferenceAttribute || "id";
-        const valueDisplayAttribute: string = findMetaValuesAgainst?.value?.valueDisplayAttribute || "name";
+        const [ inputValue, setInputValue ] = useState("");
+        const [ claimList, setClaimList ] = useState<ValueInputAutocompleteOptionsInterface[]>([]);
+        let valueReferenceAttribute: string = findMetaValuesAgainst?.value?.valueReferenceAttribute || "id";
+        let valueDisplayAttribute: string = findMetaValuesAgainst?.value?.valueDisplayAttribute || "name";
+        const [ selectedValue, setSelectedValue ] = useState<string>(null);
```

Comment:
Let's group useState statements together.

```suggestion
        const [ inputValue, setInputValue ] = useState("");
        const [ claimList, setClaimList ] = useState<ValueInputAutocompleteOptionsInterface[]>([]);
        const [ selectedValue, setSelectedValue ] = useState<string>(null);
        
        let valueReferenceAttribute: string = findMetaValuesAgainst?.value?.valueReferenceAttribute || "id";
        let valueDisplayAttribute: string = findMetaValuesAgainst?.value?.valueDisplayAttribute || "name";
```